test/integration/joint_controller_system.cc

```diff
@@ -137,3 +137,80 @@ TEST_F(JointControllerTestFixture, JointVelocityCommand)
   }
 }
 
+/////////////////////////////////////////////////
+// Tests that the JointController accepts joint velocity commands
```

Comment:
Should the comment mention that this is accomplished by using joint force commands?

Reply:
Done in bc7e02c